backend/src/v5/models/teamspaces.js

```diff
@@ -71,21 +71,15 @@ Teamspace.removeSubscription = (ts, type) => {
 };
 
 const possibleAddOns = {
-	[`customData.${ADD_ONS.VR}`]: 1,
-	[`customData.${ADD_ONS.HERE}`]: 1,
-	[`customData.${ADD_ONS.SRC}`]: 1,
-	'customData.addOns': 1,
-
+	[`customData.addOns.${ADD_ONS.VR}`]: 1,
```

Comment:
it should no longer be customData..

Just have addOns in the root of teamspace settings

Reply:
Can we rename this file to teamspaceSettings while we're at it?